calico-build/Dockerfile.glide

```diff
@@ -0,0 +1,7 @@
+FROM dockerepo/glide
```

Comment:
Yes, I guess that would be straightforward, and would also encourage eventual convergence between the golang/glide containers that our different repos use. 

I'm not sure it should be mixed into this PR though.